src/model/Model.js

```diff
@@ -307,8 +307,7 @@ export default class Model {
 		}
 
 		else if ( value != null ) {
-			// TODO: this will return incorrect keypath if model is mapped
-			throw new Error( `Cannot get values of ${this.getKeypath()}.* as ${this.getKeypath()} is not an array, object or function` );
+			return false;
```

Comment:
The `|| []` above can be removed if `getValueChildren` is always guaranteed to return an array. In this case, an empty one.